tests/test_trigger_no_status/results_true.dat

```diff
@@ -1,27 +1,28 @@
 k-combined:
 9.809303E-01 7.264435E-03
-tallies:
+tally 1:
```

Comment:
Same numbers, different order.  Do you mind that they are in a different order?  I certainly haven't changed any OpenMC physics, and I think the test is still effective.  If you disagree, I could make a class specialization for the trigger tests.